supervisor/api/jobs.py

```diff
@@ -0,0 +1,37 @@
+"""Init file for Supervisor Jobs RESTful API."""
+import logging
+from typing import Any, Dict
+
+from aiohttp import web
+import voluptuous as vol
+
+from ..coresys import CoreSysAttributes
+from ..jobs.const import ATTR_IGNORE_CONDITIONS, JobCondition
+from .utils import api_process, api_validate
+
+_LOGGER: logging.Logger = logging.getLogger(__name__)
+
+SCHEMA_OPTIONS = vol.Schema(
+    {vol.Optional(ATTR_IGNORE_CONDITIONS): [vol.Coerce(JobCondition)]}
+)
+
+
+class APIJobs(CoreSysAttributes):
+    """Handle RESTful API for OS functions."""
+
+    @api_process
+    async def info(self, request: web.Request) -> Dict[str, Any]:
+        """Return JobManager information."""
+        return {
+            ATTR_IGNORE_CONDITIONS: self.sys_jobs.ignore_conditions,
```

Comment:
Would be nice to include `self.sys_jobs.jobs` here as well since we are adding an endpoint for /info

Reply:
I agree, but that part feels not ready to expose. We should tweak the format like `module.function` and support sub-jobs and all that stuff. After that, we know in which format we want to expose it without breaking change on the next release